src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -41,6 +41,11 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
   private videoMaxBandwidthKbps: number = DefaultDeviceController.defaultVideoMaxBandwidthKbps;
 
   private useWebAudio: boolean = false;
+  private deviceIdGroupIdMap: { [mediaDeviceKind: string]: Map<string, string> | null } = {
```

Comment:
From `Map<string, string> | null`, `| null` is not required since it's always be `new Map()`.

Reply:
done